.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: trusty
```

Comment:
I add this to prevent the fail of installing Oracle JDK. But we may consider using OpenJDK instead.